[DOC] New procedure for config of external clients in deployment guide (#3147)
strimzi/strimzi-kafka-operator: documentation/assemblies/deploying/assembly-deploy-verify.adoc

// deploying/master.adoc

[id="deploy-verify_{context}"]
= Setting up client access to the Kafka cluster

> Having xref:deploy-tasks_{context}[deployed Strimzi], the procedures in this section show how to:

Reviewer: ```suggestion
After you have xref:deploy-tasks_{context}[deployed Strimzi], the procedures in this section explain how to:
```

* Deploy example producer and consumer clients, which you can use to verify your deployment

> * How to set up external client access to the Kafka cluster

Reviewer: ```suggestion
* Set up external client access to the Kafka cluster
```

> +

Reviewer: Remove the `+`.

Reviewer: In fact, The sentence on line 13 confused me a little. Is it related to the second bullet point?

Author: It is related. So I'm going to leave it with the bullet.

> The steps to set up access to the Kafka cluster for a client outside Kubernetes are a lot more involved,

Reviewer: Not sure "involved" is friendly for localization. Maybe we can say "more complex" instead. 

"The steps to set up access to the Kafka cluster for a client outside Kubernetes are more involved..."

Author: Good point. I've changed.

> and require familiarity with the process of link:{BookURLUsing}#assembly-deployment-configuration-str[configuring Kafka components^].

Reviewer: ```suggestion
and require familiarity with link:{BookURLUsing}#assembly-deployment-configuration-str[how to configure Kafka components^].
```
